Add phone calls & available models (#26)
fixie-ai/ultradox: docs/src/content/docs/api/calls.mdx

        <th class="w-10/12"></th>
      </tr>
      <tr>
        <td class="font-mono">initator</td>
        <td>string</td>

>         <td>Who was responsible for staring this call. Typically set to agent for outgoing calls and left as the default (user) otherwise.<br />Can be `"agent"` or `"user"` (default).</td>

Reviewer: ```suggestion
        <td>Who was responsible for staring this call. Typically set to INITIATOR_AGENT for outgoing calls and left as the default (INITIATOR_USER) otherwise.</td>
```

      </tr>
      <tr>
        <td class="font-mono">languageHint</td>
        <td>string</td>
        <td>A hint about the language to be used if your end users are speaking a language other than English. Must be a valid BCP47 language code. Best effort is made to select the closest supported language.  (default: "en")</td>
      </tr>
      <tr>
        <td class="font-mono">maxDuration</td>
        <td>string</td>

>         <td>Maximum length for the call in the format `HH:MM:SS`. Can be used to limit the length of the call. Optionally, set `timeExceededMessage` to have the agent say a message when the allotted time is reached.<br /><br />For the free plan, defaults to amount of free time remaining (less any calls that are currently in-progress). For paid plans, defaults to the system maximum of one hour ("01:00:00"). </td>

Reviewer: Please update format.  Should be number of seconds (possibly fractional) followed by "s".

Author: done
